<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1073</width>
    <height>734</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>20</y>
      <width>451</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>STATISTICS AND REPORTS</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>501</width>
      <height>581</height>
     </rect>
    </property>
    <property name="title">
     <string> Revenue Statistics</string>
    </property>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>481</width>
       <height>90</height>
      </rect>
     </property>
     <property name="title">
      <string>Quarterly Revenue Statistics</string>
     </property>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>55</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Quarterly</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cbxQuarterly">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>40</y>
        <width>101</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Quarterly 1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Quarterly 2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Quarterly 3</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Quarterly 4</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>245</x>
        <y>40</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Year</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cbxYear">
      <property name="geometry">
       <rect>
        <x>284</x>
        <y>40</y>
        <width>110</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btnFilterByQuarterly">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>40</y>
        <width>61</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Filter</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>481</width>
       <height>90</height>
      </rect>
     </property>
     <property name="title">
      <string>Monthly Revenue Statistics</string>
     </property>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Month       From</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateFromByRevenue">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>36</y>
        <width>110</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>40</y>
        <width>21</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>To</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateToByRevenue">
      <property name="geometry">
       <rect>
        <x>287</x>
        <y>36</y>
        <width>110</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btnFilterByMonthRevenue">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>33</y>
        <width>61</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Filter</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>250</y>
       <width>481</width>
       <height>261</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QTableWidget" name="tblInvoice">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>55</y>
        <width>441</width>
        <height>192</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btnReloadTableOrder">
      <property name="geometry">
       <rect>
        <x>368</x>
        <y>13</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Reload</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>535</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Total Revenue</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtTotalRevenue">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>530</y>
       <width>147</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>100</y>
      <width>521</width>
      <height>581</height>
     </rect>
    </property>
    <property name="title">
     <string>Product Statistics</string>
    </property>
    <widget class="QGroupBox" name="groupBox_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>39</y>
       <width>491</width>
       <height>90</height>
      </rect>
     </property>
     <property name="title">
      <string>Statistics By Category</string>
     </property>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>40</y>
        <width>55</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Category</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cbxCategory">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>40</y>
        <width>191</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btnSearch">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>37</y>
        <width>81</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Search</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>491</width>
       <height>90</height>
      </rect>
     </property>
     <property name="title">
      <string>Monthly Product Statistics</string>
     </property>
     <widget class="QPushButton" name="btnFilterByMonthProduct">
      <property name="geometry">
       <rect>
        <x>413</x>
        <y>34</y>
        <width>61</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Filter</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>263</x>
        <y>40</y>
        <width>21</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>To</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>33</x>
        <y>40</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Month       From</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateFromByProduct">
      <property name="geometry">
       <rect>
        <x>133</x>
        <y>36</y>
        <width>110</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateToByProduct">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>36</y>
        <width>110</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_8">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>250</y>
       <width>491</width>
       <height>261</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QTableWidget" name="tblProduct">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>55</y>
        <width>451</width>
        <height>192</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Bestseller</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtBestSeller">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>16</y>
        <width>171</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btnReloadTableProduct">
      <property name="geometry">
       <rect>
        <x>375</x>
        <y>13</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Reload</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>193</x>
       <y>535</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Total quantity sold</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtTotalQuantitySold">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>530</y>
       <width>171</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="txtUserName">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>20</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1073</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
